### PR TITLE
YOTJ 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Intruder's Inquisition
+ManaCost:B R
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ DBDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature an opponent controls.
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | RememberTargets$ True | TgtPrompt$ Select target creature an opponent controls | AILogic$ PowerDmg | NumDmg$ ParentTargeted$CardPower | ExcessSVar$ Excess | DamageSource$ ParentTarget | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | ConditionCheckSVar$ Excess | ConditionSVarCompare$ GE1 | Defined$ RememberedController | NumCards$ 1 | DiscardValid$ Card.nonLand+cmcEQX | Mode$ TgtChoose | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup
+SVar:X:Count$ValidHand Card.RememberedController$GreatestCMC
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt:8]: This isn't correct. I find it hard to make this work with to different targetedcontrollers

File: forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Intruder's Inquisition
+ManaCost:B R
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ DBDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature an opponent controls.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt:4]: Description incomplete

File: forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt:8]: just use ControlledBy to access full Defined power

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt]: Any pointers ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt]: `ControlledBy ParentTarget`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt]: Got it working this way thanks

File: forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | AILogic$ PowerDmg | NumDmg$ ParentTargeted$CardPower | ExcessSVar$ Excess | DamageSource$ ParentTarget | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | ConditionCheckSVar$ Excess | ConditionSVarCompare$ GE1 | Defined$ ParentTargetedController | NumCards$ 1 | DiscardValid$ Card.nonLand+cmcEQX | Mode$ TgtChoose | SpellDescription$ If excess damage was dealt to a creature this way, its controller discards a card with the greatest mana value among cards in their hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt:6]: why nonLand?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**